CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## 5.3.3 (Next)
 
-* Your contribution here.
+* [#128](https://github.com/radar/distance_of_time_in_words/pull/128): Improves backwards compatibility with the original Rails `#distance_of_time_in_words` helper by adding support for numeric values for both the `from_time` and `to_time` arguments - [@bjedrocha](https://github.com/bjedrocha)
```

Comment:
I would just say "Adds support ...", we usually don't explain why, just say what we did.